src/helpers/mod.rs

```diff
@@ -20,8 +20,10 @@ pub fn get_file_type() -> &'static str {
 pub fn get_platform_name() -> &'static str {
     if cfg!(target_os = "windows") {
         "nvim-win64"
-    } else if cfg!(target_os = "macos") {
-        "nvim-macos"
+    } else if cfg!(target_os = "macos") && cfg!(target_arch = "arm64") {
```

Comment:
```suggestion
    } else if cfg!(target_os = "macos") && cfg!(target_arch = "aarch64") {
```

https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch

Reply:
merged the other suggestion lmao